tests/test_annotations.py

```diff
@@ -262,19 +263,45 @@ def test_merge(project):
     input_annotations = input_annotations[
         input_annotations["set"].isin(["vtc_rttm", "alice"])
     ]
+    print(input_annotations)
     am.import_annotations(input_annotations)
     am.read()
 
+    print(am.annotations)
     am.read()
     am.merge_sets(
         left_set="vtc_rttm",
         right_set="alice",
         left_columns=["speaker_type"],
         right_columns=["phonemes", "syllables", "words"],
         output_set="alice_vtc",
+        full_set_merge = False,
+        recording_filter = {'sound.wav'}
     )
     am.read()
 
+    anns = am.annotations[am.annotations['set'] == 'alice_vtc']
+    assert anns.shape[0] == 1
+    assert anns.iloc[0]['recording_filename'] == 'sound.wav'
+    
+    time.sleep(2) #sleeping for 2 seconds to have different 'imported_at' values so that can make sure both merge did fine
```

Comment:
nice trick!

Reply:
The automated tests are fasr =/ . I need to make sure the importation was indeed only done on the second file and skipped the existing one. But this does not look like an elegant solution. I will try to think about a better one